packages/guideline-blocks-settings/src/hooks/useAttachments.spec.tsx

```diff
@@ -77,3 +77,28 @@ describe('useAttachments', () => {
         });
     });
 });
+
+describe('useAttachmentsContext', () => {
+    it('should throw an error when not a child of a provider', async () => {
+        expect(() => renderHook(() => useAttachmentsContext())).toThrowError();
+    });
+
+    it('should return correct info', async () => {
```

Comment:
Maybe it's worth to add a test for the HOC as well

Reply:
fixed